### PR TITLE
Move changelog entries of backports for 6.3.1

### PR DESCRIPTION
**Pull request by jsoriano**
Move anything that there wasn't in 6.3.0 to 6.3.1. Moved also some entries that were incorrectly placed in 6.3.0.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -89,22 +87,46 @@ https://github.com/elastic/beats/compare/v6.3.0...v6.3.1[View commits]
...
 *Packetbeat*
 
 - Fix an out of bounds access in HTTP parser caused by malformed request. {pull}6997[6997]
 - Fix missing type for `http.response.body` field. {pull}7169[7169]
 
+== Added ==
+
+*Filebeat*
+
+- Collect accumulated docker network metrics and mark old ones as deprecated. {pull}7253[7253]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This should be under Metricbeat

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -115,25 +137,17 @@ https://github.com/elastic/beats/compare/v6.2.3...v6.3.0[View commits]
...
 - Rename `beat.cpu.*.time metrics` to `beat.cpu.*.time.ms`. {pull}6449[6449]
-- Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
-- Allow override of dynamic template `match_mapping_type` for fields with object_type. {pull}6691[6691]
 - Add `host.name` field to all events, to avoid mapping conflicts. This could be breaking Logstash configs if you rely on the `host` field being a string. {pull}7051[7051]
 
 *Filebeat*
 
 - Remove the undefined `username` option from the Redis input and clarify the documentation. {pull}6662[6662]
 - Add validation for Stdin, when Filebeat is configured with Stdin and any other inputs, Filebeat
+- Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This is not a breaking change I think so we can move it to `Added`

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -115,25 +137,17 @@ https://github.com/elastic/beats/compare/v6.2.3...v6.3.0[View commits]
 
 - De dot keys of labels and annotations in kubernetes meta processors to prevent collisions. {pull}6203[6203]
 - Rename `beat.cpu.*.time metrics` to `beat.cpu.*.time.ms`. {pull}6449[6449]
-- Mark `system.syslog.message` and `system.auth.message` as `text` instead of `keyword`. {pull}6589[6589]
-- Allow override of dynamic template `match_mapping_type` for fields with object_type. {pull}6691[6691]
 - Add `host.name` field to all events, to avoid mapping conflicts. This could be breaking Logstash configs if you rely on the `host` field being a string. {pull}7051[7051]
 
 *Filebeat*
 
 - Remove the undefined `username` option from the Redis input and clarify the documentation. {pull}6662[6662]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: This is also not a breaking change as the feature never existed I think.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (jsoriano) [reply CHANGELOG.asciidoc]: Fixed

File: CHANGELOG.asciidoc (hunk 3 above)
Comment (jsoriano) [reply CHANGELOG.asciidoc]: Moved to Added section.

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (jsoriano) [reply CHANGELOG.asciidoc]: I also think this is not breaking, but this is explicitly mentioned as breaking in the PR https://github.com/elastic/beats/pull/6589, and maybe we should mark all changes in existing fields as breaking. Leaving it here by now.

**Pull request merged**